For some reason, this PR is failing CI tests here. I see this in the log, but not clear why because this PR is not changing that:
```
Collecting tables
--
159 | Downloading https://files.pythonhosted.org/packages/d2/55/fb95c1c82750f05eb0fd082cc95c25f56cc32a4fd3518794cce1914af1ff/tables-3.7.0.tar.gz (8.2MB)
160 | Installing build dependencies: started
161 | Installing build dependencies: finished with status 'done'
162 | Getting requirements to build wheel: started
163 | Getting requirements to build wheel: finished with status 'error'
164 | ERROR: Command errored out with exit status 1:
165 | command: /opt/python/bin/python3.8 /opt/python/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpogu40kll
166 | cwd: /tmp/pip-install-40suiqv8/tables
167 | Complete output (14 lines):
168 | /tmp/H5close2gxp8cvt.c: In function ‘main’:
169 | /tmp/H5close2gxp8cvt.c:2:5: warning: implicit declaration of function ‘H5close’ [-Wimplicit-function-declaration]
170 | H5close();
171 | ^~~~~~~
172 | /usr/bin/ld: cannot find -lhdf5
173 | collect2: error: ld returned 1 exit status
174 | cpuinfo failed, assuming no CPU features: No module named 'cpuinfo'
175 | * Using Python 3.8.3rc1 (default, May  1 2020, 20:57:46)
176 | * Found cython 0.29.28
177 | * USE_PKGCONFIG: True
178 | .. ERROR:: Could not find a local HDF5 installation.
179 | You may need to explicitly state where your local HDF5 headers and
180 | library can be found by setting the ``HDF5_DIR`` environment
181 | variable or by using the ``--hdf5`` command-line option.
182 | ----------------------------------------
183 | ERROR: Command errored out with exit status 1: /opt/python/bin/python3.8 /opt/python/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpogu40kll Check the logs for full command output.
184 | WARNING: You are using pip version 19.2.3, however version 22.0.4 is available.
185 | You should consider upgrading via the 'pip install --upgrade pip' command.
186 | make: *** [Makefile:11: aws_lambda] Error 1
```

--- a/build/Makefile
+++ b/build/Makefile
@@ -8,7 +8,7 @@ install: egg
 
 aws_lambda:
 	rm -Rf artifact_build && mkdir artifact_build
-	cd artifact_build; pip3 install -t . chardet requests urllib3 certifi idna pandas pytz six pyyaml matplotlib pyparsing cycler kiwisolver numexpr mock tables
+	cd artifact_build; pip3 install -t . chardet==3.0.4 requests==2.23.0 urllib3==1.25.9 certifi==2020.4.5.1 idna==2.9 pandas==1.0.3 pytz==2020.1 six==1.14.0 pyyaml==5.3.1 matplotlib==3.2.1 pyparsing==2.4.7 cycler==0.10.0 kiwisolver==1.2.0 numexpr==2.7.1 Pillow==7.1.2 tables==3.6.1
 	cd artifact_build; rm -Rf *.dist-info __pycache__ bin setuptools
 	cd artifact_build; pwd
 	cd artifact_build; cp -R ../../python/src/fsoi .

